_data/projects/quickperf.yml

```
@@ -8,7 +8,7 @@ tags:
 - performance
 - quickperf
 upforgrabs:
-  name: ":thumbsup: good first issue"
+  name: good first issue
```

Reverting this to it's old value is what's required here:

```suggestion
  name: ":thumbsup: good first issue"
```